com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/ConsoleProgress.java

```diff
@@ -96,7 +97,14 @@ public void done() {
 
     @Override
     public boolean isCanceled() {
```

Comment:
We already have this checks in many places, I think it makes sense to reuse it to avoid new entities which will do almost the same thing